indispensable-asn1/src/jvmTest/kotlin/at/asitplus/signum/indispensable/asn1/Asn1ParserTest.kt

```diff
@@ -67,10 +72,13 @@ class Asn1ParserTest : FreeSpec({
             repeat(9) { source.readAsn1Element().first shouldBe childIterator.next() }
 
 
-            shouldThrow<Asn1Exception> { Asn1Element.parseAll(withGarbage.iterator()) shouldBe seq.children }
+            shouldThrow<Asn1Exception> { run {
+                withGarbage.iterator()
+                source.readFullyToAsn1Elements()
+            } shouldBe seq.children }
```

Comment:
this doesn't look right? are you discarding the return value of `withGarbage.iterator()`?